cmd/kubelet/kubelet.go

```diff
@@ -45,12 +45,18 @@ func die(err error) {
 }
 
 func main() {
```

Comment:
I agree that it could be better. @liggitt and I wanted to hoist constructing the configuration outside of Run (since Run takes KubeletServer as an argument, it didn't really make sense to reconstruct it in Run when loading from a file or checkpoint), and main is the only level above Run. 

It may be worth adding some methods to KubeletServer to make the flow more clear, though I have to think about the cleanest way to do this.